src/index.html

```diff
@@ -5,8 +5,360 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptop&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link"> Video</a>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card-container">
+        <article class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac&quot; Retina 5k Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__product">Product code: 195434</p>
+
+          <div class="card__review">
+            <ul class="card__review-stars stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__review-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-text">
+            <p class="card__price-value">Price:</p>
+
+            <p class="card__money">$2,199</p>
+          </div>
```

Comment:
I would change class names a bit
```suggestion
          <div class="card__price">
            <p class="card__price-text">Price:</p>

            <p class="card__proce-value">$2,199</p>
          </div>
```